benches/plonk.rs

```diff
@@ -85,7 +85,7 @@ fn constraint_system_benchmark(c: &mut Criterion) {
 
     let rng = &mut rand_core::OsRng;
     let label = b"dusk-network";
-    let pp = PublicParameters::setup(1 << final_degree, rng)
+    let pp = PublicParameters::setup(1 << (final_degree - 1), rng)
```

Comment:
why is this required for a bug fix?

Reply:
We are reverting it, this is the original code, that we had to modify in https://github.com/dusk-network/plonk/commit/8efda59b0c07821c4a1ee66f7172143326b28b48 to avoid a crash, related with the same.

Reply:
Because you are never using so many public parameters, as per how the benchmark loop iterating over all the possible circuit degrees was implemented. The maximum circuit degree benchmark is final degree - 1.

Reply:
care to write it as a comment ;)